Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:latest
+WORKDIR /etc/
+RUN mkdir -p /etc/Sphinx/build
+
+RUN apk add --no-cache python3 py3-pip make git
+RUN pip3 install --upgrade pip && \
+    pip3 install git+https://github.com/sphinx-doc/sphinx && \
+    pip3 install sphinx-autobuild && \
+    pip3 install sphinx-rtd-theme
+
+CMD sphinx-autobuild -b html --host 0.0.0.0 --port 80 /etc/Sphinx/source /etc/Sphinx/build
```

Comment:
Nitpick: you're already in `/etc/` so you may be able to use shorter relative paths here and in line 3.

Reply:
Done :)